packages/scheduler/src/forks/Scheduler.js

```diff
@@ -606,9 +583,6 @@ export {
   unstable_getCurrentPriorityLevel,
   shouldYieldToHost as unstable_shouldYield,
   requestPaint as unstable_requestPaint,
-  unstable_continueExecution,
-  unstable_pauseExecution,
-  unstable_getFirstCallbackNode,
```

Comment:
idk if it's ok to delete these methods or if we should keep them around as noops, seems like something you would have an opinion on @acdlite